editor/project_manager.cpp

```diff
@@ -143,7 +143,7 @@ class ProjectDialog : public ConfirmationDialog {
 			install_status_rect->set_texture(new_icon);
 		}
 
-		set_size(Size2(500, 0) * EDSCALE);
+		set_size(Size2i(500, 500) * EDSCALE);
```

Comment:
Probably don't need to fix this, as Window now properly computes the minimum size and enforces it.